lib/rubocop/cop/ezcater/graphql_fields_naming.rb

```diff
@@ -66,7 +66,7 @@ def method_name(node)
         end
 
         def message(style)
-          format(MSG, style: style)
+          format(MSG, style:)
```

Comment:
Will this change be backwards compatible with older versions of ruby that we still support (such as 2.7)?

Reply:
Yes, indeed. I reverted and added a Rubocop disable to get rid of the warning.